yalafi/packages/cleveref.py

```diff
@@ -0,0 +1,246 @@
+
+#
+#   YaLafi module for LaTeX package cleveref
+#   Contributed by @torik42 (at GitHub) in pull request #171
+#
+
+from yalafi.defs import InitModule, Macro, VoidToken
+from yalafi import utils
+import re
+
+
+require_packages = []
+
+macro_read_sed = '\\YYCleverefInput'
+
+# Regular Expressions used to read sed file:
+re_ref = re.compile(r'''
+s/\\
+(                   # group 1: '\cref' or '\Cref'
+    \\cref|\\Cref
+)
+(?:\\)?             # non capturing group
+(\*?)               # group 2: '*' if present, '' else
+\{
+    ([^\}\{]+)      # group 3: the label of the reference

```

Comment:
Just a small feedback. Inside of `[]`, the braces `{}` do not have special meaning, as is the case for the dot `.`. I have removed the redundant backslashes in branch `master`.

Reply:
Okay. I also tried to remove the redundant backslashes wherever it should be possible and run into a problem somewhere. I can’t remember exactly. Thus, I decided to keep them everywhere.